docs/modules/ROOT/pages/references/quality-requirements/performance/portal-api-data-freshness.adoc

```diff
@@ -0,0 +1,27 @@
+= Portal API Data Freshness
+
+Source::
+Portal API client
+
+Stimulus::
+Portal API client reads data from Portal API
+
+Environment::
+{product} Portal API client
+
+Artifact::
+Portal API client and Portal API
+
+Response::
+Portal API returns the requested data
+
+Response measure::
+The read data is at most 10 minutes older than the original data in the backend system (source-of-truth)
```

Comment:
I do not see a need for such a short duration. Depending on the data, I might also be OK with data that is 24 hours old.

Consider the payment status of an invoice that is supposed to be payed through bank transfer. The Banks most likley will provide updated data once every day. So what is the deal with aiming for a 10 minutes freschness when there is nothing to refresh?

There might be data where freshness makes a difference, but I disagree with phrashing it in such a general way.

Reply:
This requirement talks about the freshness between the backend system and the API, not what and when the backend system retrieves third-party external data.

Reply:
Let me phrase it differently. With that requirement, an implementation has to run at least every 10 minutes. That is a waste of resources when it is known that there won't be any changes because the backend system gets updated only once a day.

Reply:
No that I have read the wording more carefully, it is about the time delta a change in the backend gets reflected in the API. But then again, why? It could be that we have cases where a short delta is important. For the examples I have in mind, it does not matter and would even make things more complicated.

Reply:
@tobru  and I did briefly discuss this in a call today about that. We realized that we need to split this requirement into 2. Since "data" is too generic we misunderstood its impact.
- One requirement deals with the entities like User, Organization or Teams that gets synced to _all zones_. There we need faster sync intervals.
- For invoices and other externally-managed entities, which are only in _appuio global_ we can loosen the restriction to $tbd hours or so.